cms/api.py

```diff
@@ -37,6 +36,8 @@
 from cms.utils.plugins import copy_plugins_to_placeholder
 from menus.menu_pool import menu_pool
 
+from six import string_types
```

Comment:
I am not sure why python 2 support is being kept with six?

Reply:
Django 3.0 is also redundant 

This is a backport of the original change for django 3.0. This has been done to have a direct comparison to the original change. There is another PR for django 3.1 and another for 3.2 which is a few PRs combined.